examples/networking-gre-playbook/site.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Configure GRE Tunnel between rtr1 and rtr2
+  hosts: routers
+  gather_facts: no
+  connection: local
+  vars:
+     rtr1_public_ip: "34.236.147.137"
```

Comment:
Where do these IPs come from? If someone from the community wants to use/try this what do they need to do? This needs to be documented as a required variables or the like etc.

Reply:
Good catch, I fix that.